sequences/__main__.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import pandas as pd
+from keras import Sequential
+from keras.layers import LSTM, Dense
+from matplotlib import pyplot
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.feature_selection import SelectFromModel
+from sklearn.metrics import accuracy_score, confusion_matrix
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler
+
+
+def predict_sequence():
+    sample = pd.read_csv("../seq/sample.csv")
+    # ressources = pd.read_csv("../seq/res.csv")
+    sample = prepare_data(sample)
+
+    # create series and split in training and validation sets
+    x_series, y_series = create_series(sample, 5, 1)
+    x_train, x_validate, y_train, y_validate = train_test_split(x_series, y_series, test_size=0.2)
+
+    print("x_train", x_train.shape)
+    print("y_train", y_train.shape)
+    print("x_validate", x_validate.shape)
+    print("y_validate", y_validate.shape)
+
+    # Now we can train and check our model
+    model = train_model(x_train, y_train, x_validate, y_validate)
+
+    check_model(model, x_train, y_train, name="Training set")
+    check_model(model, x_validate, y_validate, name="Validation set")
+
+
+def prepare_data(data):
+    """
+    Prepare the data before training
+    :param data: raw data
+    :return: prepared data
+    """
+    data = data.drop("event_id", axis=1)
+    data = data.drop("timestamp", axis=1)
```

Comment:
Depending on the model, you might want to keep the timestamp but in this case that's fine